azure-pipelines/build.yml

```diff
@@ -69,13 +87,35 @@ steps:
     command: restore
     projects: test/Microsoft.SqlTools.ServiceLayer.UnitTests
 
+- task: DotNetCoreCLI@1
+  displayName: 'dotnet restore test/Microsoft.Kusto.ServiceLayer.UnitTests'
+  inputs:
+    command: restore
+    projects: test/Microsoft.Kusto.ServiceLayer.UnitTests
+
 - task: DotNetCoreCLI@1
   displayName: 'dotnet test test/Microsoft.SqlTools.ServiceLayer.UnitTests'
   inputs:
     command: test
     projects: test/Microsoft.SqlTools.ServiceLayer.UnitTests
     arguments: '--logger "trx;LogFileName=xunit.trx"'
 
+- task: DotNetCoreCLI@1
+  displayName: 'dotnet test test/Microsoft.Kusto.ServiceLayer.UnitTests'
+  inputs:
+    command: test
+    projects: test/Microsoft.Kusto.ServiceLayer.UnitTests
+    arguments: '--logger "trx;LogFileName=xunit.trx"'
+
+- task: CopyFiles@2
+  displayName: 'Copy Kusto Service Layer binaries'
```

Comment:
You should just add Kusto as a project reference to STS if you're going to do this I would say.

But first - how big are these binaries? We want to avoid adding in a bunch of extra stuff that most people won't be using if we can avoid it.

Reply:
Only 3 net files get copied over. The projects are almost identical as far as dependencies except for Kusto dlls, so the copying just skips over most of them.

Reply:
Yes, but by doing this you make it so that if anyone tries to build/deploy from anything else without also doing this step they're not going to get the right results.

Ideally one should be able to easily build from a local enlistment and get the same result as in the lab. Manually piecing together deployments like this is likely to cause confusion and issues in the future.

Reply:
And you should still run this change by all the major stakeholders (and provide exact DLL sizes) before doing a change like this - we should avoid unnecessary bloat when possible.

Reply:
Got it, makes sense. Will change it up. I switched this back to draft, I will roll that suggestion at the following PR #1057